hangman/lib/hangman/server.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Hangman.Server do
+  alias Hangman.Game, as: Game
+
+  use GenServer
+
+   def init(args \\ []) do
+      { :ok, args }
+   end
+
+    def handle_call({ :new_game }, _from, _state ) do
+       { :reply, Game.new_game(), Game.new_game() }
```

Comment:
# -5 

Two problems here. First, you're calling ame.new_game twice, so the one you return isn't the one that you store in state. Second, the correct place to call new_game is in init, so it can set the state up for the server.